onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -1215,9 +1215,7 @@ TEST(ReductionOpTest, ArgMin_int32) {
                           {0, 0,
                            0, 0});
 
-  std::unordered_set<std::string> excluded_eps；
-  excluded_eps.insert(kNGraphExecutionProvider); // NGraph EP cannot handle negative axis values
-
+  std::unordered_set<std::string> excluded_eps = {kNGraphExecutionProvider}; // NGraph EP cannot handle negative axis values
```

Comment:
NGraph cannot process `axis == 0` ? 

If so, maybe this comment can be modified to reflect this ?

Reply:
Sorry - seems like this change is stale. Resolved.